concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1169,15 +1169,15 @@ Hale-1973-1798-1167	Human_Mind_and_Conditions-55	god	n	10a.55	1944	GOD
 Hale-1973-1798-1168	Human_Mind_and_Conditions-56	witch	n	10a.56	824	WITCH
 Hale-1973-1798-1169	Human_Mind_and_Conditions-57	shaman	n	10a.57	394	SHAMAN (FOLK HEALER)
 Hale-1973-1798-1170	Human_Mind_and_Conditions-58	mask	n	10a.58	2005	MASK
-Hale-1973-1798-1171	Human_Mind_and_Conditions-59	incense	n	10a.59		
+Hale-1973-1798-1171	Human_Mind_and_Conditions-59	incense	n	10a.59	3356	INCENSE
 Hale-1973-1798-1172	Human_Mind_and_Conditions-60	temple	n	10a.60	911	TEMPLE
 Hale-1973-1798-1173	Human_Mind_and_Conditions-61	idol	n	10a.61	1945	IDOL
 Hale-1973-1798-1174	Human_Mind_and_Conditions-62	faith	n	10a.62		
 Hale-1973-1798-1175	Human_Mind_and_Conditions-63	miracle	n	10a.63	2541	MIRACLE
 Hale-1973-1798-1176	Human_Mind_and_Conditions-64	prayer	n	10a.64		
 Hale-1973-1798-1177	Human_Mind_and_Conditions-65	prayer flag	n	10a.65		
 Hale-1973-1798-1178	Human_Mind_and_Conditions-66	oath	n	10a.66	1712	OATH
-Hale-1973-1798-1179	Human_Mind_and_Conditions-67	worship	n	10a.67		
+Hale-1973-1798-1179	Human_Mind_and_Conditions-67	worship	n	10a.67	257	WORSHIP
```

Comment:
`257 WORSHIP` is a verb in concepticon, so would unmap. https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L258